src/Faker/Provider/fr_FR/Address.php

```diff
@@ -44,13 +44,26 @@ class Address extends \Faker\Provider\Address
     );
 
     private static $regions = array(
+        // Les treize régions métropolitaines, par ordre alphabétique de leurs noms et codes ISO 3166-2 :
+        'Auvergne-Rhône-Alpes', 'Bourgogne-Franche-Comté', 'Bretagne', 'Centre-Val de Loire', 'Corse',
+        'Grand Est', 'Hauts-de-France', 'Île-de-France', 'Normandie', 'Nouvelle-Aquitaine', 'Occitanie', 
+        'Pays de la Loire', 'Provence-Alpes-Côte d\'Azur', 
+
+        // Les cinq régions ultramarines, par ordre alphabétique des codes ISO 3166-2 :
+        'Guadeloupe', 'Guyane (française)', 'Martinique', 'La Réunion', 'Mayotte',
+
+        // Autres régions
+        'Nouvelle-Calédonie', 'Polynésie française', 'Saint-Barthélémy', 'Saint-Martin', 'Saint-Pierre-et-Miquelon',
+        'Wallis-et-Futuna'
+    );
+        /*
```

Comment:
We're using git, so use `git diff` to see the difference between commits. You can safely remove the regions in the comments.

Reply:
Sorry, but I don't understand what the problem is?